src/Elm/Syntax/DestructurPattern.elm

```diff
@@ -0,0 +1,181 @@
+module Elm.Syntax.DestructurPattern exposing
```

Comment:
Is this name correct? (I'm not a native English speaker).

The Elm documentation uses 'destructure' instead of 'destructur' (https://elm-lang.org/docs/records).

Reply:
You're right. That makes two times I've misspelled this word. I'll fix it now.